server/src/test/java/io/spine/server/BoundedContextTest.java

```diff
@@ -181,15 +195,54 @@ void propagateRepositoriesToStand() {
         assertThat(stand.getExposedTypes(), contains(repository.getEntityStateType()));
     }
 
-    @Test
-    @DisplayName("not allow two aggregate repositories with aggregates of same state")
-    void throwOnSameAggregateState() {
-        ProjectAggregateRepository repository = new ProjectAggregateRepository();
+    @SuppressWarnings("unchecked")
+    @ParameterizedTest
+    @MethodSource("sameStateRepositories")
+    @DisplayName("not allow two entity repositories with entities of same state")
+    void throwOnSameEntityState(Pair<Repository, Repository> sameStateRepositories) {
+        Repository repository = sameStateRepositories.getKey();
         boundedContext.register(repository);
+        Repository anotherRepository = sameStateRepositories.getValue();
+        assertThrows(IllegalStateException.class, () -> boundedContext.register(anotherRepository));
+    }
 
-        AnotherProjectAggregateRepository anotherRepo = new AnotherProjectAggregateRepository();
+    /**
+     * To check whether {@link io.spine.server.BoundedContext} really throws an
```

Comment:
```suggestion
     * To check whether a {@link io.spine.server.BoundedContext} really throws
```

Please move `an` to the next line, in order not to split the article and the noun.